recipe/meta.yaml

```
@@ -1,4 +1,4 @@
-{% set version = "2.5.1" %}
+{% set version = "2.5.1rc1" %}
```

I have a preference for making the `rc` bits a separate version component (but it works either way).
```suggestion
{% set version = "2.5.1.rc1" %}
```